### PR TITLE
Added Tutorial for InterProScan

### PR DESCRIPTION
**Pull request by katarinaavucic**
A short tutorial on how to use InterProScan for Virus Discovery.

### CODE REVIEW

File: tools/InterProScan.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+**Tutorial Objective**: By the end of this tutorial, you will be able to use InterProScan to identify potential protein domains in a given protein sequence.
+
+## Input / Prerequisites
+- [Tool Weblink](https://www.ebi.ac.uk/interpro/result/InterProScan/)
+- [Tutorial Data](img/template/ao.fa)
+- In order to use InterProScan to analyse your virus, you need a protein sequence of the region you would like to analyse. For this tutorial, we will be using the protein sequence containing the RdRp (RNA-dependent RNA polymerase). The protein sequence used in this tutorial is provided in the dropbox, but if you would like to try with your own virus the steps are as follows:
```
Comment (ababaian) [inline tools/InterProScan.md:11]: in the dropbox?

File: tools/img/InterProScan/potential_narnavirus_ORF.fa [hunk 2]
```diff
@@ -0,0 +1,16 @@
+>lcl|ORF4:66:3164 unnamed protein product
```
Comment (ababaian) [inline tools/img/InterProScan/potential_narnavirus_ORF.fa]: Could you give this a more descriptive sequence name please?

**Review by ababaian — COMMENTED** [review]
Great Tutorial :) There's a couple of minor points if you could fix them.

File: tools/InterProScan.md (hunk 1 above)
Comment (katarinaavucic) [reply tools/InterProScan.md:11]: Updated this so now the path leads to the file

**Comment by katarinaavucic** [comment]
Sorry, I did not see these requests until an hour and a half ago! I accidentally pushed my A3 onto this branch as well, so let me know if you need me to change anything.

File: tools/img/.DS_Store
Comment (ababaian) [inline tools/img/.DS_Store:1]: Please remove all dot files from PR

File: virus/.DS_Store
Comment (ababaian) [inline virus/.DS_Store:1]: remove dot file plz

File: virus/data/a3_references.bib
Comment (ababaian) [inline virus/data/a3_references.bib:1]: For all data and images for your file, please move them to directory
`viruses/img/p_apiscarba/<p_apiscarba_references.bib>` or `.png` or whatever respective file it is.

File: virus/data/img/bioproject_table.jpg
Comment (ababaian) [inline virus/data/img/bioproject_table.jpg:1]: Beautiful use of a table, it's such a neat way to organize this information. Well done.

Comment (ababaian) [reply virus/data/img/bioproject_table.jpg:1]: Just move the path please to organize data

File: virus/Pupavirus_apiscarab.html
Comment (ababaian) [inline virus/Pupavirus_apiscarab.html:1]: HTML page not needed, keep everything in Markdown please.

File: virus/Pupavirus_apiscarab.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
+---
+bibliography: data/a3_references.bib
+nocite: '@*'
+---
+
+# A Viral Metamorphosis: Novel Virus Pupavirus apiscarab Found in Honeybee Populations in China
+written by: [Katarina Vucic](https://github.com/yourname *optional link*)
```
Comment (ababaian) [inline virus/Pupavirus_apiscarab.md]: Add link to your github.

File: virus/Pupavirus_apiscarab.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+The other Apis cerana sample was from a study by the same group that involved looking at how circRNAs
+are involved in A. cerana larval response to A. apis (BioProject PRJNA560730). In this experiment, 
+their test sample had the virus while their control run did not, which is what we would expect to see. 
+Since we know for sure that the control sample had the virus, we wanted to see if it also had traces 
+of the fungus that might have gone undetected. Using STAT SRA, we found trace amounts of Ascosphaeraceae 
+and Narnaviridae in the Apis cerana SRRs (Table 2). We also found that there seems to be a corelation
+between the amount of Ascosphaeraceae and Narnavidae in the sample. 
+```
+
+Table 2: Counts of Ascosphaeraceae and Narnaviridae reads in Apis cerana treatment and control samples
```
Comment (ababaian) [inline virus/Pupavirus_apiscarab.md:160]: Fix the image link paths.

File: virus/Pupavirus_apiscarab.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+```
+Finally, P. apiscarab was found in a run with Panicum virgatum (switch grass) and Tupaia belangeri
+(tree shrew). We wanted to determine if this virus could be found in a specific microbiome or if 
+these were due to contamination. To do this, we ran MGnify Search on contig 1, however no significant
+results were generated [Richardson, 2023]. Thus, the inclusion of the virus in this sample was likely
+due to some sort of contamination.
+```
+
+
+### Virus Genome {Q3}
```
Comment (ababaian) [inline virus/Pupavirus_apiscarab.md]: Improve title, I'd also recommend for your report to move the genome up higher and present it first. It's a great overview (include which SRR it's from) and will make all the subsequent analysis easier to understand/make sense when you talk RdRp and such.

File: virus/Pupavirus_apiscarab.md [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+on their wings. Much like a secret breeze, they traversed from the West to the East, spreading their
+enchantment far and wide. 
+
+This tale became a whispered warning among the village youth. They learned to discern the true nature 
+of visitors, for not every fluttering wing carried the song of a friend. And so, the children held onto 
+the magic of caution, ensuring they remained vigilant against the beguiling enchantment of the fairies 
+who sought to steal their light.
+```
+
+## References
```
Comment (ababaian) [inline virus/Pupavirus_apiscarab.md]: Include a link to your bib file here.

**Review by ababaian — CHANGES_REQUESTED** [review]
Can you make the `virus` data and image files consistent with how it's organized in the `tools` tutorial please?

- `virus/Pupavirus_apiscarab.md`

- `virus/img/Pupavirus_apiscarab/<data_files>`

- `virus/img/Pupavirus_apiscarab/<image_files>`

**Comment by katarinaavucic** [comment]
Yes, sorry! I pushed that by accident I am still working on the rest of my changes :).

**Comment by katarinaavucic** [comment]
Sorry for the delay, I was trying to incorporate the image without changing too much of my original submission. Let me know if there is anything else that needs to be fixed!

**Review by ababaian — APPROVED** [review]

**Pull request merged**